pkg/unpackerr/handlers.go

```diff
@@ -194,7 +194,7 @@ func (u *Unpackerr) handleXtractrCallback(resp *xtractr.Response) {
 		u.Printf("Extraction Started: %s, items in queue: %d", resp.X.Name, resp.Queued)
 		u.updateQueueStatus(&newStatus{Name: resp.X.Name, Status: EXTRACTING, Resp: resp}, true)
 	case resp.Error != nil:
-		u.Printf("Extraction Error: %s: %v", resp.X.Name, resp.Error)
+		u.Errorf("Extraction Error: %s: %v", resp.X.Name, resp.Error)
```

Comment:
Since you're here, please make this say `Extraction Failed`. This reduces the redundancy of the word error.